### ISSUE TITLE
Filter by category

### ISSUE THREAD

**Issue by vvsagar** [dagmawibabi/ScholarXIV#41]
**Is your feature request related to a problem? Please describe.**
I really like the app, but I only care about certain category of papers (physics:hep-ex in my case) so I would appreciate it if I can just see those papers on the homepage when I open sorted by date.

**Describe the solution you'd like**
Maybe in settings, I can select the arxiv categories I'd like to be shown on homepage?

**Comment by Cooper-J2000** [comment]
Yes, filtering articles by category and sorting them by time is a pretty good feature！

**Comment by dagmawibabi** [comment]
@Cooper-J2000 we're working on the category, but sorting by date is already implemented

**Comment by Cooper-J2000** [comment]
Thank you very much for your work！

**Comment by Cooper-J2000** [comment]
@dagmawibabi Thank you very much for your work

**Comment by dagmawibabi** [comment]
@vvsagar @Cooper-J2000 please checkout ScholarXIV.com and let me know what you think of it :)

**Comment by vvsagar** [comment]
@dagmawibabi it looks great! Thank you 🙇🏽
